packages/ui/src/components/ui/slider/slider.tsx

```diff
@@ -0,0 +1,53 @@
+"use client"
+
+// SPDX-FileCopyrightText: 2024 Deutsche Telekom AG
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { sliderVariants } from "@pixelshades/styles/components/slider"
+import { RenderSlot } from "@pixelshades/utils/jsx"
+import type { SliderProps as AriaSliderProps } from "react-aria-components"
+import {
+	Label as AriaLabel,
+	Slider as AriaSlider,
+	SliderOutput as AriaSliderOutput,
+	SliderTrack as AriaSliderTrack,
+	SliderThumb as AriaSliderThumb,
+} from "react-aria-components"
+import type { VariantProps } from "tailwind-variants"
+import { If } from "../../utils"
+
+const { sliderRoot, sliderHeader, sliderTrack, sliderThumb, sliderDescription } = sliderVariants()
+
+interface SliderProps extends AriaSliderProps, VariantProps<typeof sliderVariants> {
+	className?: string
+	label?: string
+	description?: React.ReactElement<HTMLElement>
+}
+
+const Slider = ({ className, label, description, orientation, ...props }: SliderProps) => {
```

Comment:
Slider should use the Label Component aswell for consistency and tooltip support. See Textfield